### DISCUSSION TITLE
Alarm Notfications?

### DISCUSSION THREAD

**Discussion by hl2guide** [Windos/BurntToast#174] [Q&A]
I'm trying to use the alarm notification, but for me it just defaults to 9 minutes and does not use _ExpirationTime_.

e.g.

```
$expirationTime = (Get-Date).AddMinutes(20)
New-BurntToastNotification -Text 'example alarm' -Sound 'Alarm2' -SnoozeAndDismiss -ExpirationTime $expirationTime
```

Am I missing something, are there official docs or a wiki?

**Comment by Windos** [comment]
Hey @hl2guide, sorry I missed this discussion come through!

I do need to do a better job on the docs here, it's way up on my to do list... when I manage to carve out some free time again 😅

The expiration time there is "The time after which the notification is no longer relevant and should be removed from the Action Center."

Think of it like you're sending a reminder that there will be an event happening in the staff lunchroom at 1 PM. You could set the expiration time to 1:15 so that Windows will automatically remove it as having it persist could cause confusion (like seeing it the following day and thinking the event was happening then).

The nine-minute default is... annoying. The docs indicate it should be 10, but Windows has been deciding to make that 9 for a few years now and I haven't found out why.

For more info on customizing that, check out: https://toastit.dev/2019/11/21/crouton-13-hitting-snooze/

**Comment by Windos** [comment]
Hello, and thank you for your contribution to BurntToast.

Today marks the official release of [BurntToast v1.0.0](https://github.com/Windos/BurntToast/releases/tag/v1.0.0). As part of this major milestone and to ensure the project starts its next chapter with a clean slate, I am declaring "issue bankruptcy."

This decision is explained in more detail in the official announcement post: [Ten Years Toasting: BurntToast Hits the Big v1!](https://toastit.dev/2025/07/22/burnttoast-v1/)

As a result, I am closing all outstanding issues and pull requests (and discussions), including this one. Please do not interpret this as a lack of interest in the problem, suggestion, or topic you've raised. This is a necessary step to clear a very old backlog and focus development efforts on the current state of the module.

**If this topic is still relevant to you:**

1.  Please **open a new discussion**. You are welcome to link back to this one for historical context.

I hope this reset will allow me to be more responsive to community feedback going forward.

Thank you for your understanding and for being part of the BurntToast journey.
